src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -51,6 +53,9 @@ public class SnowflakeSinkConnectorConfig {
   public static final String BEHAVIOR_ON_NULL_VALUES_CONFIG = "behavior.on.null.values";
 
   // Buffer thresholds
+  public static final String BUFFER_ENABLE_SINGLE_BUFFER = "buffer.enable.single.buffer";
```

Comment:
Is there a description on what is this used for?

Reply:
and should we prefix it with `snowflake.streaming`?

Reply:
added description and new prefix